Dizzy/Screens/PlaceStory/PlaceStoryVC.swift

```diff
@@ -43,6 +43,7 @@ final class PlaceStoryVC: ViewController {
         layoutViews()
         setupViews()
         bindViewModel()
+        view.backgroundColor = .black
```

Comment:
this line should go in to the function `setupViews()`

Reply:
done